docs/ci.md

```diff
@@ -215,3 +249,23 @@ $ pushd test
 $ make shard-test WHAT="should enforce egress policy allowing traffic to a server in a different namespace based on PodSelector and NamespaceSelector"
 $ popd
 ```
+
+To skip the IPv4 only tests (in a IPv6 only deployment), pass the
+`KIND_IPV6_SUPPORT=true` environmental variable to `make`:
+
+```
+$ cd $GOPATH/src/github.com/ovn-org/ovn-kubernetes
+
+$ pushd test
+$ KIND_IPV6_SUPPORT=true make shard-n-other
+$ popd
+```
+
+Github CI doesn´t offer IPv6 connectivity, so IPv6 only tests are always
```

Comment:
we should have a github issue to track this, just so in the future if Actions allows us to do this we can remember to fix it.